src/circuit/index.ts

```diff
@@ -47,88 +14,16 @@ export interface AutoRelayConfig {
   maxListeners: number
 }
 
-export interface RelayInit extends RelayConfig {
-  addressSorter?: AddressSorter
+export interface HopConfig {
+  enabled?: boolean
+  active?: boolean
```

Comment:
How are active and enabled different? Also, we only use `enabled` in `Relay` to decide whether or not to advertise the node as a relay.